### PR TITLE
Update v12 `color` token migration

### PR DESCRIPTION
**Pull request by lgriffee**
### WHY are these changes introduced?

While working on `color` migrations in one of the web deps I came across some `color` tokens that were in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Bm3-a-e5BRCelz1VmPJHmjrGrd-ZtFVH_b0_yFpzBHc/edit#gid=2044767004) but not in our current migrations and charts. 

### WHAT is this pull request doing?
Adds the following `color` token replacement maps to `v12-styles-replace-custom-property-color` migration and our v11 to v12 migration guide: 

| Deprecated Token                                         | Replacement Value                          |
| -------------------------------------------------------- | ------------------------------------------ |
| `--p-color-bg-interactive-active`                        | `--p-color-bg-fill-brand-active`           |
| `--p-color-bg-interactive-disabled`                      | `--p-color-bg-surface-disabled`            |
| `--p-color-bg-interactive-hover`                         | `--p-color-bg-fill-brand-hover`            |
| `--p-color-bg-interactive-subdued` | `--p-color-bg-surface-brand` |
|`--p-color-border-strong-hover` | `--p-color-border-tertiary` |

### CODE REVIEW

File: .changeset/purple-turkeys-pretend.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/polaris-migrator': minor
+'polaris.shopify.com': minor
```
Comment (aaronccasanova) [inline .changeset/purple-turkeys-pretend.md]:
```suggestion
'@shopify/polaris-migrator': patch
'polaris.shopify.com': patch
```

**Review by aaronccasanova — APPROVED** [review]

**Review by sarahill — APPROVED** [review]

**Review by chloerice — APPROVED** [review]
Thank you @lgriffee!! 🙌🏽

**Pull request merged**